lib/routes/admin-api/tag.test.js

```diff
@@ -7,26 +7,30 @@ const store = require('../../../test/fixtures/store');
 const permissions = require('../../../test/fixtures/permissions');
 const getLogger = require('../../../test/fixtures/no-logger');
 const getApp = require('../../app');
+const { createServices } = require('../../services');
+const { UPDATE_FEATURE } = require('../../permissions');
 
 const eventBus = new EventEmitter();
 
 function getSetup() {
```

Comment:
note to self: it seems like this setup function is used in multiple unit-test. Probably worth having it in a unit-test helper.

Reply:
Agreed. But out-of-scope for now.